chef/cookbooks/keystone/recipes/ha.rb

```diff
@@ -65,5 +65,12 @@
     only_if { CrowbarPacemakerHelper.is_cluster_founder?(node) }
   end
 
+  pacemaker_order "o-#{service_name}" do
+    ordering "g-haproxy g-postgresql g-rabbitmq cl-#{service_name}"
```

Comment:
That's not correct:
- it doesn't depend on haproxy
- postgresql / rabbitmq might not be on this cluster
- it doesn't depend on rabbitmq (afaik; it's just used for notifications)
- there might not be a `g-postgresql` resource (when drbd is used)